ui/pages/Staff/components/SuccessStory.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+
+import SortableTable from 'shared/components/table/SortableTable'
+import { HorizontalSpacer, VerticalSpacer } from 'shared/components/Spacers'
+import DataLoader from 'shared/components/DataLoader'
+import { InlineHeader, ColoredIcon } from 'shared/components/StyledComponents'
+import {
+  FAMILY_SUCCESS_STORY_TYPE_OPTIONS,
+  FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP,
+} from '../../../shared/utils/constants'
+import { SUCCESS_STORY_COLUMNS } from '../constants'
+import { loadSuccessStory } from '../reducers'
+import { getSuccessStoryLoading, getSuccessStoryLoadingError, getSuccessStoryRows } from '../selectors'
+import TagFieldView from '../../../shared/components/panel/view-fields/TagFieldView'
+
+const getDownloadFilename = successStoryTypes => `success_story_${successStoryTypes}`
+
+// eslint-disable-next-line camelcase
+const getFamilyFilterVal = ({ success_story }) => `${success_story}`
+
+const LOADING_PROPS = { inline: true }
+
+const ACTIVE_LINK_STYLE = {
+  cursor: 'notAllowed',
+  color: 'grey',
+}
+
+const formatInitialValue = (match) => {
+  const query = match.params.successStoryTypes
+  let queryToArr = []
+  if (query === 'all') {
+    queryToArr = Object.keys(FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP)
+  }
+  else if (query) {
+    queryToArr = query.split(',')
+  }
+  return { successStoryTypes: queryToArr }
+}
+
+const SuccessStory = ({ match, data, loading, loadingError, load, filters, history }) =>
+  <DataLoader contentId={match.params.successStoryTypes} load={load} reloadOnIdUpdate content loading={false}>
+    <InlineHeader size="medium" content="Types:" />
+    <TagFieldView
+      isEditable
+      editLabel="choose success story types"
+      field="successStoryTypes"
+      idField="row_id"
+      initialValues={formatInitialValue(match)}
+      tagOptions={FAMILY_SUCCESS_STORY_TYPE_OPTIONS}
+      onSubmit={value => history.push(`/staff/success_story/${value.successStoryTypes}`)}
+      showIconOnly
+      simplifiedValue
+      fieldDisplay={value => value.map(tag =>
+        <span>
+          <ColoredIcon name="stop" color={FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP[tag].color} />
+          {FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP[tag].name}
+          <HorizontalSpacer width={4} />
+        </span>)}
+    />
+    or <NavLink to="/staff/success_story/all" activeStyle={ACTIVE_LINK_STYLE}>view all success stories</NavLink>
+    <HorizontalSpacer width={20} />
+    {filters}
+    <VerticalSpacer height={15} />
+    <SortableTable
+      striped
+      collapsing
+      horizontalScroll
```

Comment:
Since there aren't so many columns, instead of doing `collapsing` with `horizontalScroll` and a `minWidth` on the columns, just leave these out and the table will automatically divide the width evenly among all columns